source/unified-test-format/tests/valid-fail/ignoreResultAndError.yml

```diff
@@ -0,0 +1,43 @@
+description: ignoreResultAndError  
+
+schemaVersion: '1.3'
+
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: true
```

Comment:
Is this relevant? I don't think it's necessary unless we're using transactions and want to test pinning.